locale/mr.js

```diff
@@ -1,6 +1,7 @@
 //! moment.js locale configuration
```

Comment:
Just delete the file locale/mr.js on your local file system. This deletion should create a git change. Commit that change and push it to your develop branch. 
Doing this should automatically update this Pull Request.